src/stream/src/task/stream_manager.rs

```diff
@@ -284,10 +289,17 @@ impl LocalStreamManager {
         Ok(())
     }
 
-    pub fn drop_actor(&self, actors: &[ActorId]) -> StreamResult<()> {
+    pub fn drop_actor(
+        &self,
+        source_mgr: &dyn SourceManager,
+        actors: &[ActorId],
+    ) -> StreamResult<()> {
         let mut core = self.core.lock();
         for id in actors {
             core.drop_actor(*id);
+            if let Some(table_id) = core.actor_tables.remove(id) {
```

Comment:
We should clear `actor_tables` and source manager in `stop_all_actors`, otherwise the recovery process will fail.

Reply:
0o0
Sorry for missing it. I will push some new commits about it later.